scripts/runfabtests.sh

```diff
@@ -564,6 +446,8 @@ case ${opt} in
 	;;
 	N) SKIP_NEG+=1
 	;;
+	M) USE_TEST_MATRIX=1
```

Comment:
It doesn't seem you've added this to the `usage` function. It's also a bit misleading since the test matrix seems to be used either way.